fgcm/fgcmStars.py

```diff
@@ -576,6 +576,10 @@ def loadStars(self, fgcmPars,
             if test.size > 0:
                 self.fgcmLog.info('Flagging %d stars as reference star outliers from previous cycles.' %
                                   (test.size))
+            test, = np.where((flagFlag[a] & objFlagDict['REFSTAR_RESERVED']) > 0)
+            if test.size > 0:
+                self.fgcmLog.info('Flagging %d stars as reference star reserved from previous cycles.' %
```

Comment:
This sentence isn't very clear. Something like 'Flagging %d stars as reference stars that were reserved from previous cycles.' (if I am correctly interpreting the original sentence) would be better.

Reply:
Changing it at this point will have to wait for the next version of fgcm, unless I go through a whole rigmarole.  But thank you for pointing this out.